python/src/equistore/operations/unique_metadata.py

```diff
@@ -90,8 +90,8 @@ def unique_metadata(
     :param names: a ``str``, ``list`` of ``str``, or ``tuple`` of ``str``
         corresponding to the name(s) of the indices along the specified ``axis``
         for which the unique values should be found.
-    :param gradient_param: a ``str`` corresponding to the gradient parameter
-        name for the gradient blocks to find the unique indices for. If none
+    :param gradient: a ``str`` corresponding to the gradient parameter name for
```

Comment:
`gradient` can also be a list. We should reflect this in the docs.

Reply:
No, `names` can be a list but not `gradient`